pkgdb2/templates/list_actions.html

```diff
@@ -123,7 +123,8 @@
         </a>
       </td>
       <td>
-        {% if action.package %}
+        {% if action.package or (action.action == 'request.package' and 
+              action.status == 'Approved') %}
           <a href="{{url_for('.package_info', package=action.package.name)}}">
             {{ action.package.name }}
```

Comment:
What is the value ``action.package`` if the request is of type ``request.package``?